src/main.ts

```diff
@@ -1,9 +1,21 @@
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
+import { ValidationPipe } from '@nestjs/common';
+import { SwaggerModule, DocumentBuilder, OpenAPIObject } from '@nestjs/swagger';
 
-async function bootstrap() {
+async function bootstrap(): Promise<void> {
   const app = await NestFactory.create(AppModule);
   app.enableCors();
+  app.useGlobalPipes(new ValidationPipe()); //added this for using file upload pipe (post service createPost)
+  const config = new DocumentBuilder()
+    .setTitle('ConnectHub')
+    .setDescription('The ConnectHub API description')
```

Comment:
Copy paste? 🤔

Reply:
Yeah, that part wasn't meant to be in my commit. It was my mistake; I was just experimenting with Swagger to learn how it works.